### ISSUE TITLE
Role and Label missing , Admin issue.

### ISSUE THREAD

**Issue by ramdev1980** [percussion/percussioncms#834]
**Describe the bug**
Role and Label missing for Anchor tags.

**To Reproduce**

Go to Admin  (http://localhost:9992/cm/app/?view=workflow&site=test2&path=%2FSites%2Ftest2%2F&pathType=page)
Click on a Categories Tab.
Click on Down arrow adjacent to Publish button. The options available (Publishing to Staging etc.) have no Role and Label attributes, when inspecting them in browser.

**Expected behavior**
Role and Label should be present for Anchor tags.

**Additional context**
Navigate all the options available for similar issue and fix.
